org.opent2t.onboarding.winkhub/js/README.md

```diff
@@ -0,0 +1,5 @@
+# Wink Hub Onboarding
+Sample [Wink Hub](http://www.wink.com/) onboarding module for Open Translators to Things. May be used to discover devices connected to a Wink Hub per the 
+[org.OpenT2T.Onboarding.WinkHub.xml](https://github.com/openT2T/onboarding/blob/master/org.OpenT2T.Onboarding.WinkHub/org.OpenT2T.Onboarding.WinkHub.xml) schema.
+
+See schema definition for inputs to the onboarding method, and outputs via success and error callbacks.
```

Comment:
> and outputs via success and error callbacks [](start = 58, length = 44)

I don't see outputs defined in the manifest anymore. Are they defined somewhere else?

Should this be updated?